### PR TITLE
corrects check for timeout

### PR DESCRIPTION
**Pull request by dmikey**
fixes the broken test to check on the api body

### CODE REVIEW

File: api/install_test.go [hunk 1]
```diff
@@ -11,6 +13,7 @@ import (
 
 	"github.com/blocklessnetworking/b7s/api"
 	"github.com/blocklessnetworking/b7s/models/api/request"
+	"github.com/blocklessnetworking/b7s/models/response"
```
Comment (Maelkum) [inline api/install_test.go:16]:
```suggestion
	"github.com/blocklessnetworking/b7s/models/api/response"
```

File: api/install_test.go [hunk 2]
```diff
@@ -84,8 +87,16 @@ func TestAPI_FunctionInstall_HandlesErrors(t *testing.T) {
 		err = api.Install(ctx)
 		require.NoError(t, err)
 
-		require.Equal(t, http.StatusRequestTimeout, rec.Result().StatusCode)
+		var res response.InstallFunction;
```
Comment (Maelkum) [inline api/install_test.go:90]:
```suggestion
		var res response.InstallFunction
```

Comment (Maelkum) [reply api/install_test.go:90]: I only now noticed that in `install.go` we do:

```go
	// return inner code as body
	return ctx.JSON(200, map[string]interface{}{
		"code": strconv.Itoa(status),
	})
```

For consistency, I'd do:

```go
import (
	"github.com/blocklessnetworking/b7s/models/api/response"
)

// ...

res := response.InstallFunction{
	Code: strconv.Itoa(status),
}

return ctx.JSON(http.StatusOK, res)
```

Comment (dmikey) [reply api/install_test.go:90]: Love it, thanks for learning me some golang :)

**Pull request closed without merging**